app/client/Widgets/NothingPlayingWidget_mac.mm

```diff
@@ -9,3 +9,9 @@
     [[NSWorkspace sharedWorkspace] launchAppWithBundleIdentifier:@"com.apple.iTunes" options:NSWorkspaceLaunchDefault additionalEventParamDescriptor:nil launchIdentifier:NULL];
 }
 
+void
```

Comment:
[minor] This should be in the same line as the function name.

Reply:
I would choose to do that if this was a new file, but I think it makes more sense to follow the style already in the file.